src/assets/locales/locale-en.json

```diff
@@ -501,12 +502,63 @@
           "title": "Voting Performance",
           "txt1": "All up to date"
         },
-        "percent": "Reward %",
-        "potential": "Potential Reward"
+        "percent": "Qualified reward %",
+        "potential": "Estimated rake reward %"
```

Comment:
Should either all be capital or no capital: I.E Qualified Reward, Estimated Rake Reward
![Screen Shot 2019-09-25 at 7 46 44 PM](https://user-images.githubusercontent.com/47183489/65645286-40cd5900-dfcd-11e9-8066-5019f2afeaaf.png)

Reply:
Right, will be all uppercase